packages/dashboard/src/util/alerts.ts

```diff
@@ -1,20 +1,81 @@
-import Swal from 'sweetalert2';
-
-export const successMsg = (msg: string) => {
-  Swal.fire({
-    title: 'Done!',
-    text: !!msg ? `${msg}` : 'Successful Operation',
-    icon: 'success',
-    timer: 2000,
-    heightAuto: false,
-  });
-};
-
-export const errorMsg = (error: string) => {
-  Swal.fire({
-    title: 'Ups',
-    text: !!error ? `${error}` : 'An error has occurred',
-    icon: 'error',
-    heightAuto: false,
-  });
-};
+import { createMuiTheme } from '@material-ui/core';
+import Swal, { SweetAlertIcon, SweetAlertOptions } from 'sweetalert2';
+
+const theme = createMuiTheme();
+
+interface VerificationAlertProps {
+  title?: string;
+  body?: string;
+  icon?: SweetAlertIcon;
+  confirmButtonText?: string;
+  cancelButtonText?: string;
+  confirmCallback?: () => void;
+  cancelCallback?: () => void;
+}
+
+export class Alerts {
+  static success = (message?: string) => {
+    const swalParams = Alerts.getSuccessMsg(message);
+    return Swal.fire(swalParams);
+  };
+
+  static error = (message?: string) => {
+    const swalParams = Alerts.getErrorMsg(message);
+    return Swal.fire(swalParams);
+  };
+
+  /**
+   * Modal to warn a user about something related to an action.
+   */
+  static verification = (props: VerificationAlertProps) => {
+    const { confirmCallback, cancelCallback } = props;
+    const verificationParams = Alerts.getVerificationParams(props);
+    Swal.fire(verificationParams).then((modalState) => {
+      if (modalState.isConfirmed) {
+        !!confirmCallback && confirmCallback();
+      } else {
+        !!cancelCallback && cancelCallback();
+      }
+    });
+  };
+
+  static getSuccessMsg = (message?: string): SweetAlertOptions => {
```

Comment:
Makes sense! Thanks!

Reply:
> We should also convert this to a react component, but that is for another PR

Yeah, it's difficult to test the alerts from `sweetalert2`, since they are not rendered on the jsdom, so the functionality is been tested just on the e2e test. This can be a problem in the future so I'm thinking of replacing it with https://material-ui.com/components/dialogs/ or use this react wrapper and see if I can test it https://github.com/alex-shamshurin/sweetalert2-react.